kafka/src/test/java/io/streamzi/cloudevents/KafkaHeaderTest.java

```diff
@@ -45,16 +47,23 @@ public void testConsumerRecord() {
 
         final ConsumerRecord<String, String> record = new ConsumerRecord("topic", 0, 0, System.currentTimeMillis(), CREATE_TIME, 0L, 0, 0, "Key", "Value", requiredHeaders);
 
-        final CloudEvent cloudEvent = KafkaHeaderUtil.createFromConsumerRecord(record);
+        final CloudEvent<Map<String, String>> cloudEvent = KafkaHeaderUtil.createFromConsumerRecord(record);
 
         assertThat(cloudEvent.getData().get()).isNotNull();
-        assertThat(cloudEvent.getData().get()).isEqualTo(record);
+
+        // todo: use java9 Map.of
```

Comment:
In terms of creating a library that's useful for others, sticking to Java 8 internally might be better as users of the library may or may not be able to use a Java 9 runtime

Reply:
yeah - but it's good to add these 'obvious' things :-)